URDFtemplate joints as combination of 3 rotational

MariaLazzaroni opened the pull request:
All joints as combination of 3 rotational joints

MariaLazzaroni committed:
URDFtemplate joints as combination of 3 rotational

All joints as combination of 3 rotational joints

claudia-lat commented on Experiments/23links_human/templates/XSensModelStyle_URDFtemplate.urdf:
```diff
@@ -27,6 +27,48 @@
     <!--Chain from (2) to (7)-->
+	<link name="L5_f1">
+        <inertial>
+            <mass value="FAKEMASS"/>
```
How do you choose `FAKEMASS`?

claudia-lat commented on Experiments/23links_human/templates/XSensModelStyle_URDFtemplate.urdf:
```diff
@@ -27,6 +27,48 @@
+        <inertial>
+            <mass value="FAKEMASS"/>
+            <origin xyz="0 0 0" rpy="0 0 0" />
+            <inertia ixx="FAKEIN" iyy="FAKEIN" izz="FAKEIN" ixy="0" ixz="0" iyz="0"/>
```
Also inertia?

claudia-lat commented on Experiments/23links_human/templates/XSensModelStyle_URDFtemplate.urdf:
```diff
@@ -27,6 +27,48 @@
+        <visual>
+            <origin xyz="0 0 0" rpy="0 0 0" />
+            <geometry>
+                <box size="FAKEDIM FAKEDIM FAKEDIM"/>
```
Also size?

MariaLazzaroni committed:
Update XSensModelStyle_URDFtemplate.urdf

MariaLazzaroni committed:
Update createXsensLikeURDFmodel.m

Add an option to create an URDF model for Gazebo (masses and inertias of the fake links non-zero)

MariaLazzaroni committed:
Update XSensModelStyle_URDFtemplate.urdf

MariaLazzaroni committed:
Update main.m

Add to the function createXsensLikeURDFmodel the input 'filename' and 'GazeboModel' to save the model and to create a model for Gazebo
